crates/rnote-ui/src/contextmenu.rs

```diff
@@ -0,0 +1,68 @@
+use gtk4::{glib, prelude::*, subclass::prelude::*, CompositeTemplate, PopoverMenu, Widget};
```

Comment:
Just to be consistent with all other files: Add a `// Imports` comment above the imports as the first line